UI.PageModels/Pages/Admin/Booking/HearingAssignJudgePage.cs

```diff
@@ -84,26 +83,26 @@ private void ClickSaveJudgeButton()
         ClickElement(_saveEJudge);
     }
 
-    public void clickAddJohLink ()
+    public void ClickAddJohLink ()
     {
         ClickElement(_addJudicialOfficeHolder);
     }
-    public void EnterPanelMemberDetails(BookingPanelMemberDto panelMember)
+    public void EnterPanelMemberDetails(BookingJudiciaryParticipantDto panelMember)
     {
         AssignPanelMemberDetails(panelMember.Username, panelMember.DisplayName, panelMember.InterpreterLanguage);
         ClickSavePanelMemberButton();
         Driver.TakeScreenshotAndSave(GetType().Name, "Enter panelMember Details");
     }
 
-    private void AssignPanelMemberDetails(string PanelMemberEmail, string PanelanelMemberDisplayName,
+    private void AssignPanelMemberDetails(string panelMemberEmail, string panelanelMemberDisplayName,
```

Comment:
small typo in `panelanelMemberDisplayName`

Reply:
thanks. sorted it